### ISSUE TITLE
Vim Plugin Go To Definition Only Works On First Line of Statement

### ISSUE THREAD

**Issue by arnm** [phildawes/racer#279]
If I enter `gd` on any of the symbols in the following statement, the plugin works properly:

``` rust
io::stdin().read_line(&mut guess).ok().expect("Failed to read line");
```

However, if I split the statement into multiple lines the `gd` command will not work on any of the symbols not in the first line of the statement:

``` rust
io::stdin().read_line(&mut guess) // works here
  .ok() // does not work here
  .expect("Failed to read line"); // does not work here either
```

**Comment by phildawes** [comment]
Hi @arnm, Thanks for the bug report. I'm not sure I'll get to fixing it for a little bit because I'm working on the new type inference stuff with rustc. It might be that whatever comes out of that fixes this anyway

**Comment by arnm** [comment]
@phildawes sounds good

**Comment by Sindisil** [comment]
FWIW, I hit this issue using the Rusty Code plugin for VS Code, where doc tool tips, peek def, and jump def are all affected by this bug (racer 2.0.4, rust 1.14)

**Comment by andjo403** [comment]
fixed in #686

**Comment by jwilm** [comment]
Thanks @andjo403
